sdk/python/feast/feature_store.py

```diff
@@ -153,6 +167,76 @@ def get_historical_features(
         )
         return job
 
+    def materialize(
+        self,
+        feature_views: Optional[List[str]],
+        start_date: datetime,
+        end_date: datetime,
+    ) -> None:
+        """
+        Materialize data from the offline store into the online store.
+
+        This method loads feature data in the specified interval from either
+        the specified feature views, or all feature views if none are specified,
+        into the online store where it is available for online serving.
+
+        Args:
+            feature_views (List[str]): Optional list of feature view names. If selected, will only run
+                materialization for the specified feature views.
+            start_date (datetime): Start date of query
```

Comment:
Nitpick: `Start date of query` isn't really clear. How about `Start date for time range of data to materialize into the online store`